tests/cpp/include/test_ndarray_utils.h

```diff
@@ -0,0 +1,344 @@
+/*!
+ * Copyright (c) 2017 by Contributors
+ * \file test_utils.h
+ * \brief operator unit test utility functions
+ * \author Haibin Lin
+*/
+#ifndef TEST_NDARRAY_UTILS_H_
+#define TEST_NDARRAY_UTILS_H_
+
+#include <unistd.h>
+#include <dmlc/logging.h>
+#include <gtest/gtest.h>
+#include <mxnet/engine.h>
+#include <mxnet/ndarray.h>
+#include <cstdio>
+#include <vector>
+#include <cstdlib>
+#include <string>
+#include <map>
+#include "test_util.h"
+#include "test_op.h"
+
+namespace mxnet {
+namespace test {
+
+#define ROW_SPARSE_IDX_TYPE mshadow::kInt64
+
+using namespace mxnet;
+#define TEST_DTYPE float
+#define TEST_ITYPE int32_t
```

Comment:
Oh right. The tests were removed.. Should we remove all these util functions then? Since they're not being used.